layout-server/src/main/java/org/finos/vuu/layoutserver/config/MappingConfig.java

```diff
@@ -14,20 +9,10 @@
 @Configuration
 public class MappingConfig {
 
-    private final LayoutService layoutService;
-
     @Bean
     public ModelMapper modelMapper() {
         ModelMapper mapper = new ModelMapper();
 
-        mapper.typeMap(LayoutRequestDTO.class, Layout.class)
-            .addMappings(m -> m.skip(Layout::setId));
-
-        mapper.typeMap(Metadata.class, MetadataResponseDTO.class)
-            .addMappings(m -> m.map(
-                metadata -> layoutService.getLayoutByMetadataId(metadata.getId()),
-                MetadataResponseDTO::setLayoutId));
-
```

Comment:
I wasn't sure whether it could be removed altogether. I left it in as there is some Spring magic going on with the @Bean annotation which I don't fully understand

Reply:
Having the ModelMapper `@Bean` here makes the Controller easier to test so I would keep that in. A solution would be to move `@Bean
 public ModelMapper modelMapper() {
        ModelMapper mapper = new ModelMapper();
}` in our LayoutServerApplication. This eliminates the need for MappingConfig but makes the LayoutServerApplication a little more busy. What do we think of this solution vs leaving as it is?

Reply:
In that case, I'm happy to leave it as it is.